ios/Podfile

```diff
@@ -4,8 +4,8 @@
 target 'ReactNativeSampleApp' do
   # Uncomment the next line if you're using Swift or would like to use dynamic frameworks
   use_frameworks!
-  pod 'Onfido', '10.4.0', :configurations => ['Debug']
-  pod 'Onfido-Release', '10.4.0', :configurations => ['Release']
+  pod 'Onfido', '13.0.0', :configurations => ['Debug']
+  pod 'Onfido-Release', '13.0.0', :configurations => ['Release']
```

Comment:
we can change this to a single line `pod 'Onfido', '13.0.0'`. We are phasing our `Onfido-Release` soon it'd be good to get new integrators using the single compiled framework.

Reply:
updated code.